final/Code.gs

```diff
@@ -1,180 +1,159 @@
 /**
- * Responds to an ADDED_TO_SPACE event
- * in Hangouts Chat.
- *
- * @param event the event object from Hangouts Chat
+ * Responds to an ADDED_TO_SPACE event in Hangouts Chat.
+ * @param {Event} event the event object from Hangouts Chat
```

Comment:
There is no Event type defined anywhere that this corresponds to. Should this just be object instead?

Reply:
Done.